### PR TITLE
Telephony services' integration

### PR DESCRIPTION
**Pull request by saurabh-bhosale**
First part of [Telephony services' Integration](https://github.com/frappe/erpnext/issues/13012).

1. Logs incoming calls as communications
2. Captures post-call details
3. Can make outgoing calls from incoming call's communication.

**Incoming call's Communication:** 
incoming_call

**Update Post-call details like the call recording URL or call statuses field.**
ccd

**Outgoing call's Communication:** 
outgoing_call

- [ ] Documentation

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Is this under development or available in v10.1.20?

File: erpnext/docs/user/manual/en/erpnext_integration/exotel_settings.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+#### Exotel Integration Working:- 
+
+Steps:- 
+
+1. Register yourself as an user on Exotel platform.
+
+2. To have a telephony service ready for your user, create a call flow in your Call App on Exotel platform.
+
+3. A Basic Call flow is as follows:
+Greeting -> Connect -> Passthru -> Conversation ends -> Passthru
```
[the remark this answers is not in the document]
Comment (saurabh-bhosale) [reply erpnext/docs/user/manual/en/erpnext_integration/exotel_settings.md]: @ziagithub2018 It's under development, will be available in newer versions of ERPNext soon.

**Comment by Zlash65** [comment]
Was discussed over telegram, changes and modification are yet to be done.

**Comment by nabinhait** [comment]
@saurabh-bhosale Closing the PR for now. Please reopen after finishing the pending tasks.

**Comment by mhbu50** [comment]
@saurabh-bhosale Grate Work, Any Update on this?

**Pull request closed without merging**